### ISSUE TITLE
nll_loss2d_forward_kernel: block: [3,0,0], thread: [674,0,0] Assertion `t >= 0 && t < n_classes` failed.

### ISSUE THREAD

**Issue by Ashing00** [milesial/Pytorch-UNet#406]
I just follow carvana dataset to create my dataset ,only one class.
I can train carvana dataset  but when train my dataset ,i meet below issue.
I have no idea.
I use labelms to create json file and convert to mask file.
seems the n_class is wrong,but i only 1 class ,it should be the same as carvana dataset

//-------------------------------------------------------------------------------------
: nll_loss2d_forward_kernel: block: [3,0,0], thread: [674,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1634272168290/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:95: nll_loss2d_forward_kernel: block: [3,0,0], thread: [675,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1634272168290/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:95: nll_loss2d_forward_kernel: block: [3,0,0], thread: [676,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1634272168290/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:95: nll_loss2d_forward_kernel: block: [2,0,0], thread: [738,0,0] Assertion `t >= 0 && t < n_classes` failed.
/opt/conda/conda-bld/pytorch_1634272168290/work/aten/src/ATen/native/cuda/NLLLoss2d.cu:95: nll_loss2d_forward_kernel: block: [2,0,0], thread: [739,0,0  File "/home/ss/miniconda3/envs/pytorch/lib/python3.7/site-packages/torch/nn/modules/loss.py", line 1152, in forward

**Comment by Ashing00** [comment]
ok,I just know why ,now.

**Comment by liuleiBUAA** [comment]
> ok,I just know why ,now.

how to solve this problem
